src/alpine/3.12/helix/amd64/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN apk update && \
         libffi-dev \
         linux-headers \
         llvm \
+        lldb-dev \
```

Comment:
Please list this in alphabetical order with the other packages.  Same applies to the Alpine 3.13 Dockerfile.

Reply:
Done.